### PR TITLE
add video bitrate to reconfigure

### PR DESCRIPTION
**Pull request by mlfarrell**

### CODE REVIEW

File: modeling-cmds/src/def_enum.rs [hunk 1]
```diff
@@ -949,6 +949,8 @@ define_modeling_cmd_enum! {
             pub height: u32,
             /// Frames per second.
             pub fps: u32,
+            /// Video bitrate (CBR)
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:952]: Guideline for general writing is not to use acronyms until they've been defined
```suggestion
            /// Video feed's constant bitrate (CBR)
```

Comment (mlfarrell) [reply modeling-cmds/src/def_enum.rs:952]: see the engine PR on this.  and my notes about nvenc.  Default will need to be None for the time being,

Comment (mlfarrell) [reply modeling-cmds/src/def_enum.rs]: after digging for like 5 mins, I can't find my comment on that PR for the live of me.  but the gist of it was nvenc defaults the bitrate to 0 and doesn't really let us know what it's picking.  So mapping None to 0 is what i've done to re-obtain default behavior.

File: modeling-cmds/src/def_enum.rs [hunk 2]
```diff
@@ -949,6 +949,8 @@ define_modeling_cmd_enum! {
             pub height: u32,
             /// Frames per second.
             pub fps: u32,
+            /// Video feed's constant bitrate (CBR)
+            pub bitrate: Option<u32>,
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:953]:
```suggestion
            #[serde(default)]
            pub bitrate: Option<u32>,
```

**Review by adamchalmers — CHANGES_REQUESTED** [review]

**Review by adamchalmers — APPROVED** [review]

**Pull request merged**